scripts/download-zls.ps1

```diff
@@ -2,6 +2,8 @@ push-location .cache
 try {
     git clone https://github.com/zigtools/zls
     set-location zls
-    git checkout 62f17abe283bfe0ff2710c380c620a5a6e413996
+    git pull
```

Comment:
"git pull" followed by a checkout doesnt really make sense, but if this works that's fine. this whole setup needs an overhaul later

Reply:
these changes shouldn't even be part of this PR. I'll remove them